ui/ceval/src/engines/engines.ts

```diff
@@ -123,6 +126,7 @@ export class Engines {
           tech: 'HCE',
           requires: ['sharedMem', 'simd', 'dynamicImportFromWorker'],
           variants: variants.map(v => v[0]),
+          cloudEval: true,
```

Comment:
fsf14 HCE was never cached before (at least since the eval cache flush) because it never met the requirement of "NNUE". Going by https://fairy-stockfish.github.io/nnue/ I don't think we should include it either.